Lib/http/server.py

```diff
@@ -1256,10 +1254,12 @@ def test(HandlerClass=BaseHTTPRequestHandler,
 
 if __name__ == '__main__':
     import argparse
+    import contextlib
+    import functools
```

Comment:
Could you import partial only so that line 1278 doesn’t change

Reply:
Yes, done.